docs/topics_5/cli-args.adoc

```diff
@@ -37,7 +37,7 @@ To run the {ProductShortName} command without prompting, for example when execut
 |--includeTags| A space-delimited list of tags to use. When specified, only rules with these tags will be processed. To see the full list of tags, use the `--listTags` argument.
 |--input |A space-delimited list of the path to the file or directory containing one or more applications to be analyzed. This argument is required.
 |--install |Specify add-ons to install. The syntax is `<GROUP_ID>:<ARTIFACT_ID>[:<VERSION>]`. For example, `--install core-addon-x` or `--install org.example.addon:example:1.0.0`.
-|--keepWorkDirs| Flag to instruct {ProductShortName} to not delete temporary working files, such as the graph database and extracted archive files. This is useful for debugging purposes.
+|--keepWorkDirs| Flag to instruct {ProductShortName} to not delete temporary working files, such as the graph database and unzipped archives. This is useful for debugging purposes.
```

Comment:
Revert this change. I made it this morning

Reply:
Fixed.